src/Common/src/System/Net/WebSockets/WebSocketValidate.cs

```diff
@@ -0,0 +1,111 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.IO;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+
+using Microsoft.Win32;
```

Comment:
This and `System.Diagnostics.CodeAnalysis` above can probably be removed, I don't see any types from those namespaces used in the file.